doc/radosgw/s3/bucketops.rst

```diff
@@ -751,26 +751,12 @@ Parameters are XML encoded in the body of the request, in the following format:
       <TargetPrefix>string</TargetPrefix>
       <LoggingType>Standard|Journal</LoggingType>
       <ObjectRollTime>integer</ObjectRollTime>
-      <Filter>
```

Comment:
lets keep the `<Filter>` hierarchy, just keep only the `<s3Key>` as the only option inside, so we can add more types i nthe future